jwst/fits_generator/verifiers.py

```diff
@@ -95,7 +95,7 @@ def is_cds(val):
         raise ValueError(
             "Expected integer value for CDS")
 
-    day = (value & 0xffff000000000000) >> (32 + 16)
+    day = (value & 0xffff000000000000) >> (32 + 16) # noqa F841
```

Comment:
I assume this line is doing something with bit shifting so is needed

Reply:
This whole function looks wrong - `value` is not defined anywhere, and neither is `error_collector`.  I hope and trust this isn't being used, so we should probably remove it.

Reply:
is `fits_generator` used anywhere at all?

Reply:
It looks like some parts of the module are called by scripts in jwst/scripts.  I can't say if those scripts are used, but clearly, these functions in template and verifiers are not!  
@tapastro - do you know more about the fits_generator module?

Reply:
Per discussion in this morning's tag up, the fits_generator module was mostly for generating test data, pre-flight, and is no longer used.  It should be deprecated and removed, but that's beyond the scope of this PR.  For now, we can just make the minimal changes to get it past the linter.